tests/basic-test.sh

```diff
@@ -104,6 +104,12 @@ assert_streq $($OSTREE log test2-no-parent |grep '^commit' | wc -l) "1"
 echo "ok commit no parent"
 
 cd ${test_tmpdir}
+empty_rev=$($OSTREE commit -b test2-no-subject -s '' $test_tmpdir/checkout-test2-4)
+omitted_rev=$($OSTREE commit -b test2-no-subject-2 $test_tmpdir/checkout-test2-4)
+assert_streq $empty_rev $omitted_rev
```

Comment:
That's going to be racy...ostree commits include timestamps, and I suspect the test is only passing for you when the commits happen to be made in the same second.  I'd pass `--timestamp="October 21 2015"` or so like some of the prune tests.